src/Makefile

```
@@ -98,6 +98,11 @@ ARGS_32 = -define:ARCH_32
 ARGS_64 = -define:ARCH_64
 
 SHARED_SYSTEM_DRAWING_SOURCES = System.Drawing/PointSizeRectangleTypeForwarders.cs
+SHARED_RESX = $(TOP)/tools/mtouch/Errors.resx
+SHARED_DESIGNER_CS = Errors.Designer.cs
```

This is needed to generate into our build directory for src/ builds:

```suggestion
SHARED_DESIGNER_CS = $(BUILD_DIR)/common/Errors.Designer.cs
```

This is required because we have a hack where we create an API comparison between this commit and the previous commit by checking out the previous commit and building it into a different BUILD_DIR, and that process requires that all generated files honor BUILD_DIR.

This will also make your .gitignore entry redundant.